chat-server/src/app.ts

```diff
@@ -42,12 +42,20 @@ export const errorHandler: ErrorRequestHandler = (err, _req, res, _next) => {
 };
 // TODO:(DOCSP-31121) Apply to all logs in the app
 const reqHandler: RequestHandler = (req, _res, next) => {
+  const { ip } = req;
   const reqId = new ObjectId().toString();
   // Custom header specifically for a request ID. This ID will be used to track
   // logs related to the same request
   req.headers["req-id"] = reqId;
   const message = `Request for: ${req.url}`;
-  logger.info(createMessage(message, req.body, reqId));
+  logger.info(
+    createMessage({
+      message,
+      requestBody: req.body,
+      requestId: reqId,
+      ipAddress: ip,
+    })
+  );
```

Comment:
```suggestion
  const messageObject = createMessage({
     message,
     requestBody: req.body,
     requestId: reqId,
     ipAddress: ip,
   })
  logger.info(JSON.stringify(messageObject));
```

Getting a type error here because the logger expects a string - not sure why exactly since it has an `any` overload. Do we even need this log?

Reply:
hmm. not getting any errs. i think you just need to rerun `npm run build` in chat-core